pygfx/objects/_events.py

```diff
@@ -373,7 +373,7 @@ def dispatch_event(self, event: Event):
                     captured_target.release_pointer_capture(pointer_id)
             if not event.bubbles or event.cancelled or target is self:
                 break
-            target = target.parent or self
+            target = getattr(target, "parent", None) or self
```

Comment:
```suggestion
target = getattr(target, "parent", self)
```

Reply:
I've thought about this before committing, but it ain't the same ;)